java/ql/src/semmle/code/java/frameworks/javase/WebSocket.qll

```diff
@@ -0,0 +1,17 @@
+import java
```

Comment:
This needs a file-level qldoc along the lines of `/** Provides classes for reasoning about ...`.